src/utils/math.ts

```diff
@@ -0,0 +1,6 @@
+export const median = (raw: number[]) => {
+  const arr = [...raw].sort();
+  const mid = Math.floor(arr.length / 2),
+    nums = [...arr].sort((a, b) => a - b);
```

Comment:
Here `arr` can be changed to `raw` and then `arr` variable can be replaced to `nums`, don't?

Reply:
We don't need this line. You're right

Reply:
We don't need this line. You're right